packages/app/src/systems/Home/components/QRCode/QRCode.tsx

```diff
@@ -0,0 +1,61 @@
+import { cssObj } from '@fuel-ui/css';
+import { Button, Card, Flex } from '@fuel-ui/react';
+import { useCallback } from 'react';
+import QRCode from 'react-qr-code';
+
+type Props = {
+  account: string;
```

Comment:
change prop name to address.

as we use account in other places refferring to Account type, it can be confusing